guides/v2.0/cloud/configure/cloud-vcl-custom-snippets.md

```diff
@@ -19,64 +19,70 @@ To create and upload these VCL snippets, you use a terminal application. You do
 
 You need the following information to create VCL snippets:
 
-* Fastly Service ID for Staging and Production to assign the snippets to a specific service or environment
-* Fastly API key used for the `FASTLY_API_TOKEN` in the commands
+-   Fastly Service ID for Staging and Production to assign the snippets to a specific service or environment
+-   Fastly API key used for the `FASTLY_API_TOKEN` in the commands
 
 To get started, review the following sections:
 
-* [Understand VCL snippet values](#vcl-curl): Provides an overview of values for Fastly VCL JSON.
-* [Prepare to create VCL snippets](#prepare): You only need to do this once!
-* [The custom VCL snippet process](#process): Walks you through the entire process, including links to custom VCL snippets you can create with ease.
+-   [Understand VCL snippet values](#vcl-curl)—Provides an overview of values for Fastly VCL JSON.
+-   [Prepare to create VCL snippets](#prepare)—You only need to do this once!
```

Comment:
The **Prepare to create VCL snippets** link no longer works because you deleted that section. Delete the link.